README.md

```diff
@@ -99,11 +84,11 @@ metadata:
 spec:
   channel: alpha
   name: jaeger
-  source: installed-upstream-community-operators
-  sourceNamespace: local-operators
+  source: upstream-community-operators
+  sourceNamespace: marketplace
 ```
 
-For OLM to act on your subscription please note that an [`OperatorGroup`](https://github.com/operator-framework/operator-lifecycle-manager/blob/274df58592c2ffd1d8ea56156c73c7746f57efc0/Documentation/design/architecture.md#operator-group-design) that matches the [`InstallMode(s)`](https://github.com/operator-framework/operator-lifecycle-manager/blob/274df58592c2ffd1d8ea56156c73c7746f57efc0/Documentation/design/building-your-csv.md#operator-metadata) in your [`CSV`](https://github.com/operator-framework/operator-lifecycle-manager/blob/274df58592c2ffd1d8ea56156c73c7746f57efc0/Documentation/design/building-your-csv.md#what-is-a-cluster-service-version-csv) needs to be present in the subscription namespace.
+Note, that in the example above, `local-operators` is a namespace that OLM is watching. For OLM to act on your subscription please note that an [`OperatorGroup`](https://github.com/operator-framework/operator-lifecycle-manager/blob/274df58592c2ffd1d8ea56156c73c7746f57efc0/Documentation/design/architecture.md#operator-group-design) that matches the [`InstallMode(s)`](https://github.com/operator-framework/operator-lifecycle-manager/blob/274df58592c2ffd1d8ea56156c73c7746f57efc0/Documentation/design/building-your-csv.md#operator-metadata) in your [`CSV`](https://github.com/operator-framework/operator-lifecycle-manager/blob/274df58592c2ffd1d8ea56156c73c7746f57efc0/Documentation/design/building-your-csv.md#what-is-a-cluster-service-version-csv) needs to be present in the subscription namespace (here `local-operators`).
```

Comment:
I don't think you need to mention `local-operators` is a namespace that OLM is watching but we might want to call out that `marketplace` needs to be a global OLM namespace. Without that the `Subscription` won't get picked up. So we might need to rethink how our stuff is deployed for upstream.

Reply:
Good point. Is there a way to query whether a namespace is a global OLM namespace?

Reply:
If this is possible, on start up we could check this and just set the status to failing if this isn't the case

Reply:
Or rather mention that the `Subscription` needs to be in the `marketplace` namespace